web/src/main/webapp/WEB-INF/pages/application_state.jsp

```diff
@@ -106,53 +133,156 @@
+                          <p class="form-control-static <c:if test='${stat.first}'>stateData</c:if>" id="reviewerNameP">${data.reviewerName}</p>
+                          <c:if test="${stat.first}">
+                            <input type="text" class="stateInput" id="reviwerNameInput" style="display:none" value="${data.reviewerName}"
+                              data-bind="valueWithInit: 'reviewerName'"
+                              data-error="<spring:message code='statehistory.error.reviewerName.length'/>"
+                              pattern="^.{3,100}$" required />
+                          </c:if>
+                        </div>
+                        <div class="help-block with-errors"></div>
+                      </div>
+                      <div class="form-group">
+                        <label for="recommendedPositionLevelInput"  class="control-label col-sm-4"><spring:message code="statehistory.field.recommendedPositionLevel"/></label>
+                        <div class="col-sm-8">
+                          <p class="form-control-static <c:if test='${stat.first}'>stateData</c:if>" id="reviewerNameP">L${data.recommendedPositionLevel}</p>
+                          <c:if test="${stat.first}">
+                              <select required class="stateInput" id="recommendedPositionLevelInput"
+                                data-bind="valueWithInit: 'recommendedPositionLevel'"
+                                data-error="<spring:message code='statehistory.error.recommendedPositionLevel.range'/>"
+                                style="display:none;">
+                                  <option disabled <c:if test="${data.recommendedPositionLevel eq null}"> selected="selected" </c:if>>
+                                                                          <spring:message code="common.pleaseChoose"/></option>
+                                  <c:forEach begin="0" end="5" step="1" var="index">
+                                      <option value="${index}" <c:if
+                                              test="${index eq data.recommendedPositionLevel}"> selected="selected" </c:if>>${index}</option>
+                                  </c:forEach>
+                              </select>
+                              </select>
+                          </c:if>
+                        </div>
+                        <div class="help-block with-errors"></div>
+                      </div>
+                  </c:when>
+
+                  <c:when test ="${data.stateName == 'tech'}">
+                     <div class="form-group">
+                        <label for="recommendedPositionLevelInput"  class="control-label col-sm-4"><spring:message code="statehistory.field.recommendedPositionLevel"/></label>
+                        <div class="col-sm-8">
+                          <p class="form-control-static <c:if test='${stat.first}'>stateData</c:if>" id="reviewerNameP">L${data.recommendedPositionLevel}</p>
```

Comment:
reviewerNameP id is used multiple places, also the id is misleading as the paragraph displays recommended position level